lib/constants.ts

```diff
@@ -29,3 +29,10 @@ export const TAGS = {
 export const HIDDEN_PRODUCT_TAG = 'nextjs-frontend-hidden';
 export const DEFAULT_OPTION = 'Default Title';
 export const SHOPIFY_GRAPHQL_API_ENDPOINT = '/api/2023-01/graphql.json';
+
+export const BASE_URL =
```

Comment:
I think we should be able to use `VERCEL_PROJECT_PRODUCTION_URL` here now.

https://vercel.com/docs/projects/environment-variables/system-environment-variables#system-environment-variables

Reply:
Bump on this! Are you still interested in taking this PR?